common/persistence/sql/sqlplugin/interfaces.go

```diff
@@ -503,192 +504,192 @@ type (
 
 	// tableCRUD defines the API for interacting with the database tables
 	tableCRUD interface {
```

Comment:
Not a blocker: Is there something that we can remember to put this in the next release notes? This is already a public API. Any plugin implementation will get compliing errors.